react/providers/api/provider.tsx

```diff
@@ -1,30 +1,39 @@
-import React, { useEffect, useReducer } from 'react'
-import { useChains } from '../../hooks/mod.ts'
-import { APIContext } from './context.ts'
-import { apiProvidersReducer } from './reducer.ts'
-import { useConfig } from '../../mod.ts'
+import React, { useEffect, useReducer } from "react";
```

Comment:
Nit: it looks like your IDE formatter is set to something other than Deno. Deno should be the formatter. I think this will fix the issue where `'` is converted to `"`